assets/src/scss/layout/_country-selector.scss

```diff
@@ -0,0 +1,195 @@
+
+#country-selector {
+  --country-selector-- {
+    color: $white;
+    font-size: $font-size-sm;
+  }
+  margin-bottom: 24px;
+  text-align: start;
+  width: 100%;
+
+  a --country-selector--a-- {
+    color: $white;
+  }
+
+  a:hover,
+  button:hover {
+    text-decoration: var(--link--hover--text-decoration, underline);
+  }
+
+  button:focus {
+    border-radius: var(--link--focus--border-radius, 4px);
+    outline: var(--link--focus--border, 2px solid rgba(255, 255, 255, 0.4));
+    outline-offset: 0;
+  }
+
+  button:focus:not(:focus-visible) {
+    outline: 0;
+  }
+
+  .country-selector-toggle-container {
+    --country-selector--toggle-container-- {
+      border-bottom: 1px solid rgba(255, 255, 255, 0.5);
```

Comment:
I don't see this necessary to implement cause I'm not sure if the editors want to edit the border-bottom